python/game.py

```diff
@@ -1,4 +1,14 @@
+import os
+
+# terminal clear command 
+def CLEAR(): # Author Avideb
+    os.system('clear')
```

Comment:
`clear` command doesn't work on `windows cmd`

Reply:
Okay we will add this in future